src/components/Comments/NewCommentForm.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  FC, FormEvent, useState, useEffect,
+} from 'react';
+import classNames from 'classnames';
+import { TextField } from './TextField';
+import { CommentData, Field, FieldIcon } from '../../types';
+import { useCommentsContext } from '../../hooks/useCommentsContext';
+import { useGlobalContext } from '../../hooks/useGlobalContext';
```

Comment:
add `index.ts` file to `hooks` folder. exports all hooks and import here with 1 import. The same for `api`